src/LocateUri.cxx

```diff
@@ -121,18 +119,19 @@ LocateUri(UriPluginKind kind,
 				     , storage
 #endif
 				     );
-	} else if (PathTraitsUTF8::IsAbsolute(uri))
+	}
+
+	if (PathTraitsUTF8::IsAbsolute(uri))
 		return LocateFileUri(uri, client
 #ifdef ENABLE_DATABASE
 				     , storage
 #endif
 				     );
-	else if (uri_has_scheme(uri))
+	if (uri_has_scheme(uri))
 		return LocateAbsoluteUri(kind, uri
 #ifdef ENABLE_DATABASE
 					 , storage
 #endif
 					 );
-	else
```

Comment:
Removing all the `else`s has nothing to do with braced init list conversion. Please don't hide changes like this one in unrelated commits. I don't agree with this change; while the `else` is redundant here, it still demonstrates to the human reader that this function consists of many alternatives; the control flow is made visible. And it is more robust this way; one missing `return` in one branch will be complained about by the compiler.